src/main/java/lotto/util/LottoManager.java

```diff
@@ -0,0 +1,143 @@
+package lotto.util;
+
+import static lotto.numbers.Lotto.MAX_LOTTO_NUMBER;
+import static lotto.numbers.Lotto.MIN_LOTTO_NUMBER;
+import static lotto.numbers.Lotto.SIZE_OF_LOTTO;
+import static lotto.util.Money.LOTTO_PRICE;
+
+import camp.nextstep.edu.missionutils.Console;
+import camp.nextstep.edu.missionutils.Randoms;
+import java.text.DecimalFormat;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import lotto.numbers.Lotto;
+import lotto.numbers.UserInputNumbers;
+
+public class LottoManager {
+
+  public static final int PERCENTAGE_FACTOR = 100;
+
+  public void runLotto() {
+    Money money = receivePurchaseAmount();
+    List<Lotto> lottos = generateLottos(money.getAmount());
+    printGeneratedLottoNumbers(lottos);
+    UserInputNumbers receivedLotto = receiveLottoNumber();
+    Map<WinningCheck, Integer> result = calculateWinningResult(lottos, receivedLotto);
+    printWinningResult(result);
+    printProfitRate(money.getAmount(), result);
+  }
+
+  private Money receivePurchaseAmount() {
+    Money money;
+    while (true) {
+      try {
+        System.out.println("구입금액을 입력해 주세요.");
+        String input = Console.readLine();
+        money = new Money(input);
+        break;
+      } catch (IllegalArgumentException e) {
+        System.out.println(e.getMessage());
+      }
+    }
+    return money;
+  }
+
+  private List<Lotto> generateLottos(int money) {
+    int numberOfLotto = money / LOTTO_PRICE;
+    System.out.println("\n" + numberOfLotto + "개를 구매했습니다.");
+
+    List<Lotto> lottos = new ArrayList<>();
+
+    for (int i = 0; i < numberOfLotto; i++) {
+      List<Integer> lottoNumbers =
+          Randoms.pickUniqueNumbersInRange(MIN_LOTTO_NUMBER, MAX_LOTTO_NUMBER, SIZE_OF_LOTTO);
+
+      try {
+        Lotto lotto = new Lotto(lottoNumbers);
+        lottos.add(lotto);
+      } catch (IllegalArgumentException e) {
+        System.out.println(e.getMessage());
+        i--;
+      }
+    }
+
+    return lottos;
+  }
+
+  private void printGeneratedLottoNumbers(List<Lotto> lottos) {
+    for (Lotto lotto : lottos) {
+      List<Integer> lottoNumbers = new ArrayList<>(lotto.getLottoNumbers());
+      Collections.sort(lottoNumbers);
+      System.out.println(lottoNumbers);
+    }
+  }
+
+  private UserInputNumbers receiveLottoNumber() {
+    while (true) {
+      System.out.println("\n당첨 번호를 입력해 주세요.");
+      String inputNumbers = Console.readLine();
+      try {
+        String[] numbersArray = inputNumbers.split(",");
```

Comment:
저는 보통 변수명에 자료형을 넣지 않는 것을 추천해요. [관련한 글](https://tecoble.techcourse.co.kr/post/2020-04-24-variable_naming/) 읽어보셔도 좋겠네요!
```suggestion
        String[] rawNumbers = inputNumbers.split(",");
```

Reply:
네! 쉼표를 기준으로 숫자를 분리해 배열로 만든 것이니 separatedNumbers로 바꾸는 것이 좋겠습니다.